ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml

```diff
@@ -36,6 +37,49 @@ ModalPopup {
             popup.close();
         }
     }
+
+    Control {
+        width: 124
+        height: 36
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: 24
+        anchors.horizontalCenter: parent.horizontalCenter
+        background: Rectangle {
+            anchors.fill: parent
+            radius: 34
+            color: Style.current.roundedButtonSecondaryHoveredBackgroundColor
+        }
+        contentItem: Item {
+            anchors.fill: parent
+            SVGImage {
+                id: dollarEmoji
+                width: 32
+                height: 32
+                anchors.left: parent.left
+                anchors.leftMargin: 2
+                anchors.verticalCenter: parent.verticalCenter
+                source: Qt.resolvedUrl("../../../../imports/twemoji/svg/1f911.svg")
+            }
+
+            StyledText {
+                anchors.left: dollarEmoji.right
+                anchors.leftMargin: 6
+                anchors.verticalCenter: parent.verticalCenter
+                text: "My Profile"
```

Comment:
```suggestion
                text: qsTr("My Profile")
```

Reply:
@jrainville Sure, I imagined that, however I couldn't find any other place for this, Utils sounds not relevant and not sure if I should create another component just for this single property. Please let me know if you have any suggestions!

Reply:
I think you commented on the wrong comment, but you can either put it in AppMain.qml, or ChatLayout.qml . Whatever the parent component is

Reply:
@jrainville yeah let's continue here :) I won't use dynamic scoping, it's not a good practice, so we either keep it in Constants with a TODO to move it when the whole thing is restructured or introduce a new singleton component name it "Config" or so and put it there. Let me know which of the 2 you prefer :)

Reply:
I'd prefer a new Singleton, but if you tell me it will be removed next week, then we can leave it in Constants. 
We can also  wait to see what others think. Maybe I'm the only one that has a problem haha

Reply:
@jrainville I don't think this will happen next week :) Added Config.qml, so I guess it should be good now